src/PowerShellEditorServices/Language/AstOperations.cs

```diff
@@ -320,12 +320,12 @@ static private bool IsPowerShellDataFileAstNode(dynamic node, Type[] levelAstMap
         /// <summary>
         /// Finds all files dot sourced in a script
         /// </summary>
-        /// <param name="scriptAst">The abstract syntax tree of the given script</param>
+        /// <param name="scriptFile">The script file to use to find dot sourced files.</param>
         /// <returns></returns>
-        static public string[] FindDotSourcedIncludes(Ast scriptAst)
+        static public string[] FindDotSourcedIncludes(ScriptFile scriptFile)
```

Comment:
Is the whole `ScriptFile` needed here? It doesn't look like the method has changed what it needs.

Reply:
It used to require only the `Ast` and now it's both the `Ast` and the script path, but I guess this is `AstOperations` and not `ScriptFileOperations`... I'll split up the parameters.

Reply:
Oh I see it now! I looked and looked but didn't see it before. I think it makes sense as `ScriptFile` now that you've explained -- was just thinking before it might be leftover from an intermediate refactor